.bazelci/presubmit.yml

```diff
@@ -35,6 +35,26 @@ platforms:
     - "--"
     - "..."
     - "-//tests/docker/..."
+  windows:
+    build_flags:
+    - "--copt=-w"
+    - "--host_copt=-w"
+    build_targets:
+    # We are running the skipped targets remotely only.
+    - "--"
+    - "..."
+    - "-//docker/..."
+    - "-//skylib/..."
+    - "-//tools/..."
+    test_flags:
+    - "--copt=-w"
+    - "--host_copt=-w"
+    test_targets:
+    # We are running the skipped targets remotely only.
+    - "--"
```

Comment:
Thanks for adding this config.
Lets just build / test the specific windows tests you just created on windows